### ISSUE TITLE
bug:  Theme is applied partially

### ISSUE THREAD

**Issue by marcinjahn** [GradienceTeam/Gradience#622]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I've been trying to apply a theme called "Alpha Mac". When I select the theme in Gradience, the Gradience window becomes properly themed:

![image](https://user-images.githubusercontent.com/10273406/194896886-1178b062-da0a-401a-aa3a-d9cb04102678.png)

However, when I log out and log back in, I see that:

- window backgrounds follow the theme (not all apps though)
- window controls (close, maximize) have 3 possible states:
  - in some apps they follow the Alpha Mac theme
  - in some apps they look like Adwaita (even though the rest of the theme is applied, like a background)
  - in some apps windows controls disappear completely, although they are still clickable

The window controls issue is kinda random. A given app may not display controls at all, and after another log out - log in, the controls are visible (but in the default Adwaita style).

### To Reproduce

1. Apply Alpha Mac theme

### Expected behavior

The theme should just work across all the apps.

### OS

Fedora Linux 36 (Workstation)

### Version

0.3.1

### Installation method

Flatpak from Flathub

### Additional context

![image](https://user-images.githubusercontent.com/10273406/194897825-9e000178-a9c3-47c2-ad01-d54b077b2e08.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 0xMRTT** [comment]
Did you tried with other themes ?

**Comment by marcinjahn** [comment]
Yeah, I did. I'm actually using "Pretty Purple" all the time. I think it is not as problematic as Alpha Mac, because it does not modify the window controls. There are a few apps that not follow the Pretty Purple theme (like the Extensions Manager or Drawing). I have no idea why though.

**Comment by 0xMRTT** [comment]
Are you using Extensions Manager as a flatpak ?

**Comment by 0xMRTT** [comment]
Hmm, could you try:

```
flatpak override --reset
sudo flatpak override --filesystem=xdg-config/gtk-4.0
sudo flatpak override --filesystem=xdg-config/gtk-3.0
```

**Comment by marcinjahn** [comment]
I tried applying Alpha Mac again, and invoked the command you showed me. It didn't help with that issue unfortunately.
Here's what some of the apps look like after log-out/log-in:

![image](https://user-images.githubusercontent.com/10273406/194925942-d6fab64a-bb34-4589-8f3f-55db6e471bc3.png)

The window controls are missing in some apps, in others they are correctly displayed.
I never see such an issue with Pretty Purple.

**Comment by tfuxu** [comment]
@0xMRTT 
Please move this issue to `GradienceTeam/Community` as this is an issue with Alpha Mac preset itself.

**Comment by tfuxu** [comment]
> I tried applying Alpha Mac again, and invoked the command you showed me. It didn't help with that issue unfortunately. Here's what some of the apps look like after log-out/log-in:
> 
> ![image](https://user-images.githubusercontent.com/10273406/194925942-d6fab64a-bb34-4589-8f3f-55db6e471bc3.png)
> 
> The window controls are missing in some apps, in others they are correctly displayed. I never see such an issue with Pretty Purple.

Can confirm it, probably author of this preset just forgot to add some CSS rules for GTK 3 compatibility.

**Comment by tfuxu** [comment]
Known issue, it's something with CSS parser we are using in Gradience, it will only cause issues if you are changing something in preset, and it got loaded with those errors.

**Comment by marcinjahn** [comment]
> Known issue, it's something with CSS parser we are using in Gradience, it will only cause issues if you are changing something in preset, and it got loaded with those errors.

Not sure if that's what you meant, but I didn't change anything in the preset. I just applied the theme, didn't make any modifications to it.
